mobile/src/main/java/rs/readahead/washington/mobile/views/adapters/uwazi/UwaziTemplatesAdapter.kt

```diff
@@ -59,6 +59,9 @@ class UwaziTemplatesAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
                             ResourcesCompat.getDrawable(resources,R.drawable.star_border_24dp,null)
                         }
                     )
+                    contentDescription =
+                        if (entityRow.isFavorite) context.getString(R.string.action_unfavorite) else context.getString(
+                            R.string.action_favorite)
```

Comment:
you can utilize the ternary operator (?:) to make the code more concise  `val contentDescription = context.getString(if (entityRow.isFavorite) R.string.action_unfavorite else R.string.action_favorite)`